### DISCUSSION TITLE
[Problem] 50642457 Окна перекрывают рабочее поле.

### DISCUSSION THREAD

**Discussion by Ltybc1980** [FreeCAD/FreeCAD#11447] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

  FreeCAD 0.21.1 использовал  минимальный размер шрифтов и все равно - не в одном окне не возможно работать. Окно боковой панели занимает 1/4 экрана, панель приборов 1/2 экрана . Содержимое что бы редактировать (левой панели) надо растягивать окно на весь экран И ЭТО при минимально возможном шрифте. При максимальных настройках шрифтов - "рабочего поля" вообще нет 1/8 экрана.

### Full version info

```shell
[code]
OS: Linux Mint 20.2 (X-Cinnamon/cinnamon)
Word size of FreeCAD: 64-bit
Version: 0.21.1.33694 (Git) AppImage
Build type: Release
Branch: (HEAD detached at 0.21.1)
Hash: f6708547a9bb3f71a4aaade12109f511a72c207c
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: Russian/Russia (ru_RU)
Installed mods: 
  * Manipulator 1.5.0
[/code]
```

### Subproject(s) affected?

None

### Anything else?

нет

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by xtemp09** [comment]
Приложил бы скриншот. Откуда разработчикам знать настройки твоих шрифтов?

**Comment by Ltybc1980** [comment]
Это настройки программы, ( я только пытаюсь в ней учиться) но это реально баг, 
![Uploading Без названия.png…]()
как здесь приложить срин?

**Comment by Ltybc1980** [comment]
![Снимок экрана от 2023-11-18 22-33-08](https://github.com/FreeCAD/FreeCAD/assets/151398898/981cf9b0-ba02-4f76-b8ab-512cef714c02)
![Uploading Снимок экрана от 2023-11-18 22-32-36.png…]()

**Comment by Ltybc1980** [comment]
![Снимок экрана от 2023-11-18 22-32-36](https://github.com/FreeCAD/FreeCAD/assets/151398898/c522db2a-de6c-4d47-8309-1d5663454644)
при минимальных настройках шрифтов

**Comment by Ltybc1980** [comment]
Шрифты  в нижней левой панели вообще не поддаются настройки  и панель приходится растягивать для редактирования до максимума. Это при настройке шрифтов "малый 16 рх"
![Снимок экрана от 2023-11-18 22-47-46](https://github.com/FreeCAD/FreeCAD/assets/151398898/03d2b118-62b6-4f83-8a1a-173dc01e686a)
как то так

**Comment by Ltybc1980** [comment]
![Снимок экрана от 2023-11-18 23-00-02](https://github.com/FreeCAD/FreeCAD/assets/151398898/c9acf8fe-1070-4c19-88aa-0c9144e5967a)
1 Как минимум надо сократить в 5 раз шрифт
2 Как минимум  сократить шрифт в 3 раза
3 Рабочее поле очень мало. Сократив шрифты  в 1 и 2 получи:  поле 3 будет удовлетворительным для работы.
В данном исполнение не возможно сделать

**Comment by hasecilu** [comment]
If you press `Alt + F11` you can gain a few pixels from the title bar and the Linux Mint Panel.

**Comment by Ltybc1980** [comment]
ALt + F11 не решает проблемы: маленькой рабочей области. Ко всему прочему  на скрин показал не возможность изменять  -размер  шрифта ( выделил 1) Данная консоль, для внесения в неё параметров, требут постоянно растягивать и закрывать рабочую область, что очень не удобно. Настройками параметров невозможно решить проблемы.

**Comment by hasecilu** [comment]
I know it doesn't solve the problem but I think I could help a little bit, in this cases every pixel count.

**Comment by luzpaz** [comment]
Please use english in the bugtracker.

Converted to discussion. Looks UI/UX related. OP should use the new overlay feature for more screen 'real estate'

**Comment by xtemp09** [comment]
@Ltybc1980, попробуй запустить FreeCAD с параметром `QT_SCALE_FACTOR`



Как нибудь вот так:

```
 QT_SCALE_FACTOR=1.3 ./FreeCAD
```

меняя `QT_SCALE_FACTOR` от 1.3 до 1.7. Может быть, даже 1 будет хорош.



Если сработает, его можно установить в качестве глобальной переменной среды, как нибудь вот так:

```
export QT_SCALE_FACTOR=1.3
```

**Reply by Ltybc1980** [reply]
Я не смог (QT_SCALE_FACTOR=1.3 ./FreeCAD)  =  I couldn't (QT_SCALE_FACTOR=1.3 ./FreeCAD)

**Reply by xtemp09** [reply]
Что скрывается во фразе "я не смог"? Не влияет на приложение? Главное окно становится противным? Что даёт коэффициент 1?

**Comment by Ltybc1980** [comment]
НАШЕЛ РЕШЕНИЕ
Проблема в настройках операционной системы:  Коэффициент масштабирования текста стоял 1,5  снизив  до 1 становится комфортно работать  в FreeCad/  Но совершенно не комфортно  все стальное в Операционной системе.  ВЫнужден постоянно перестраивать на время работы в FreeCad   :-(
![Снимок экрана от 2023-11-20 06-13-12](https://github.com/FreeCAD/FreeCAD/assets/151398898/0fd40d32-65ed-4866-81b9-5eebe2d00f5b)

FOUND A SOLUTION
The problem is in connecting the system: The text scaling factor was 1.5 less than 1, and it becomes comfortable to work in FreeCad. Forced to constantly reconfigure while working in FreeCad 
 теперь стало так: 
![Снимок экрана от 2023-11-20 06-30-06](https://github.com/FreeCAD/FreeCAD/assets/151398898/956e3696-d211-4eab-a211-a2aad54b2c77)
